src/components/CustomCollection/CustomCollection.tsx

```diff
@@ -177,11 +179,7 @@ const CustomCollection = ({
         'Save CMS link',
         `${title} / ${existingLink.label || existingLink.url}`
       )
-      handleAddBookmark(
-        existingLink.url || '',
-        existingLink.label,
-        existingLink.id
-      )
+      handleAddBookmark(existingLink.url, existingLink.label, existingLink.id)
```

Comment:
I changed this because `url` is a required value on the `CMSBookmark` type, so `existingLink.url` should never be false/empty